pkg/upgrade/plan/plan.go

```diff
@@ -184,3 +186,12 @@ func sha256sum(s ...string) string {
 	}
 	return fmt.Sprintf("%x", h.Sum(nil))
 }
+
+func AppendIfMissing(slice []string, element string) []string {
```

Comment:
Is this function used outside of this package? If not, can you make it unexported?